src/summary/MedicationRangeChartVisualizer.jsx

```diff
@@ -183,91 +246,113 @@ class MedicationRangeChartVisualizer extends Component {
         const value = med.medication.amountPerDose ? med.medication.amountPerDose.value : null;
         const unit = med.medication.amountPerDose ? med.medication.amountPerDose.units : null;
         const name = med.medication.medication;
-
-        const numColsChart = this.state.medicationVisWide ? 5 : 12;
-        const numColsInfo = this.state.medicationVisWide ? 7 : 12;
         const medicationIsChange = (med.medicationChange ? true : false);
 
-        // If there is a medication change and it is of type "stop", don't render the medication chart or the table
-        if (med.medicationChange && med.medicationChange.type === 'stop') {
-            return (
-                <div key={i} className="medication-chart-item" ref={(parent) => {this.parent = parent}}>
-                    <Grid className="FullApp-content" fluid>
-                        <Row top="xs">
-                            <Col sm={numColsChart}>
+        return (
+            <div key={i} className="medication-chart-item" ref={(parent) => { this.parent = parent }}>
+                <Grid className="FullApp-content" fluid>
```

Comment:
Is the "FullApp-content" class needed here? I think it's used in other places and wanted to make sure we needed it here as well so that we don't end up with unexpected changes if the class changes.